helm-chart/templates/cvat_opa/deployment.yml

```diff
@@ -53,6 +53,18 @@ spec:
           env:
           {{- toYaml . | nindent 10 }}
           {{- end }}
+          {{- if .Values.cvat.opa.readinessProbe.enabled }}
+          readinessProbe:
+            tcpSocket:
```

Comment:
OPA has an actual [health endpoint](https://www.openpolicyagent.org/docs/latest/rest-api/#health-api), so perhaps you could use that?